examples/docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+services:
+  mojosdk:
+    image: mojosdk
+    build:
+      dockerfile: ./Dockerfile.mojosdk
+    container_name: mojosdk
+    tty: true
+    stdin_open: true
+    ports:
+      - 8888:8888
+    volumes:
+      - ../../:/mojo
+    environment:
+      - NVIDIA_VISIBLE_DEVICES=all
```

Comment:
Is there a reason you want to inject GPU devices into the container by using the ENV variable? Mojo doesn't (yet) currently support GPUs. Same comment below for the 1 GPU device you're requesting as the device resource.

Reply:
I was wondering if there was any GPU support yet.  I can update it to remove this.